live-examples/js-examples/string/string-padstart.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">var str1 = '5';
+
+console.log(str1.padStart(2, '0'));
+// expected output: '05'
+
+var last4Digits = '2034399002125581'.slice(-4);
+var maskedNumber = last4Digits.padStart(16, '*');
```

Comment:
I think it might be better to have a variable for the full number, and use its length instead of a hardcoded 16 here. That way the length is only represented in one place:

```
var fullNumber = '2034399002125581';
var last4Digits = fullNumber.slice(-4);
var maskedNumber = last4Digits.padStart(fullNumber.length, '*');
```

Reply:
Thank you so much for the comments! I can't make the fixes until late Monday or early Tuesday, but as soon as I can, I will make all of the suggested changes!